calculator.rb

```diff
@@ -0,0 +1,46 @@
+
+# Taking operation input from user
+
+def user_input
+ arr_op = ["+", "add", "-", "subtract", "*", "multiply", "/", "divide"]
+ puts "Would you like to add, subtract, multiply or divide? "
+ op1 = gets.chomp.to_s
+ while !arr_op.include?(op1)
+   puts "Please enter valid operator. #{arr_op}"
+   op1 = gets.chomp.to_s
+ end
+ return op1
+end
+
+
+#  Taking numbers from user and calculating them
+#  using operations from previous method
+
+def calculate (user_input, num1, num2)
+  puts "Please enter first number: "
```

Comment:
I like that you use a method here, but this method ends up being pretty big, especially when your operations are complex (like divide, where you need to test for 0). A good strategy here might be to split each operation out into its own method.